src/lib/components/product.svelte

```diff
@@ -46,9 +56,8 @@
     display: flex;
     flex-direction: column;
     justify-content: center;
-    transform: translate(0, 33%);
+    transform: translate(0, 32px);
```

Comment:
how did you get to this number?

Reply:
adjust it manually when inspecting

Reply:
it won't work if you change the default font size then

Reply:
I suppose this is to hide what is below the header - for now use a var, but in the future ALWAYS calculate this number